Makefile

```diff
@@ -121,6 +128,10 @@ docker-stability:
 	cd integration/stability && ./bind_mount_linux.sh
 	cd integration/stability && ./hypervisor_stability_kill_test.sh
 
+podman: ginkgo
```

Comment:
uhmm not running in parallel?

Reply:
@devimc right now, I am not contempleting in doing that